### PR TITLE
JS feature flag web demo

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

**Comment by diberry** [comment]
@AlexandraKemperMS @Muksvso

**Review by Muksvso — APPROVED** [review]
Looks good.

**Comment by zhenlan** [comment]
Thanks for the contribution, @diberry.

While this may work as a demo, it has a few issues
- Every request to this app will cause a call to App Config. I'm concerned if someone takes this code and uses it on their server. This is not something we would recommend. Caching is critical on the server.
- The example ignored the filters completely. When a feature flag has filters, its state is determined by the filters not just the enabled property.

We should also have a brief introduction of the example in https://github.com/Azure/AppConfiguration/blob/main/examples/README.md.

File: examples/JavaScript/WebDemoFeatureFlag/azure-app-configuration.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const { AppConfigurationClient } = require("@azure/app-configuration");
+
+if (!process.env.production) {
+  require("dotenv").config();
+  console.log("debug environment");
+}
+
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+console.log(connectionString)
```
Comment (zhenlan) [inline examples/JavaScript/WebDemoFeatureFlag/azure-app-configuration.js:9]: Please don't log the connection string. Not a good practice to log secrets.

Comment (diberry) [reply examples/JavaScript/WebDemoFeatureFlag/azure-app-configuration.js]: @zhenlan Removed, please check the PR now

**Comment by diberry** [comment]
@zhenlan Can you please merge this code if you approve?

**Comment by zhenlan** [comment]
Discussed offline. Due to [issues mentioned earlier](https://github.com/Azure/AppConfiguration/pull/575#issuecomment-973494353), we would not recommend customers to use code like this in any production service. Closing.

**Pull request closed without merging**